### PR TITLE
Implement Error pages

### LINKED ISSUE #140: Show 404 not found page If the Tx does not exist
Accessing a Tx page by entering a Tx that does not exist in the URL will result in a page with empty Tx information, but a 404 Not Found page should be displayed.

### PR DESCRIPTION
**Pull request by Yamaguchi**
resolves #140 

Show error page when 
- Path not found (404 page)
- Access to backend service fails

### CODE REVIEW

File: backend/actions/block_detail.js [hunk 1]
```diff
@@ -47,6 +47,7 @@ app.get('/api/block/:blockHash', async (req, res) => {
     logger.error(
       `Error retrieving information for block  - ${blockHash}. Error Message - ${err.message}`
     );
+    res.status(503).send('Service Temporary Ununavailable');
```
Comment (rantan) [inline backend/actions/block_detail.js:50]:
```suggestion
    res.status(503).send('Service Temporary Unavailable');
```

File: backend/actions/block_detail.js [hunk 2]
```diff
@@ -67,6 +68,7 @@ app.get('/api/block/:blockHash/raw', async (req, res) => {
     logger.error(
       `Error retrieving raw data for block  - ${blockHash}. Error Message - ${error.message}`
     );
+    res.status(503).send('Service Temporary Ununavailable');
```
Comment (rantan) [inline backend/actions/block_detail.js:71]:
```suggestion
    res.status(503).send('Service Temporary Unavailable');
```

File: backend/actions/block_detail.js [hunk 3]
```diff
@@ -90,5 +92,6 @@ app.get('/api/block/:blockHash/txns', async (req, res) => {
     logger.error(
       `Error retrieving txns for block  - ${blockHash}. Error Message - ${error.message}`
     );
+    res.status(503).send('Service Temporary Ununavailable');
```
Comment (rantan) [inline backend/actions/block_detail.js:95]:
```suggestion
    res.status(503).send('Service Temporary Unavailable');
```

File: backend/actions/transaction_detail.js [hunk 4]
```diff
@@ -35,6 +35,7 @@ app.get('/api/tx/:txid', async (req, res) => {
     logger.error(
       `Error retrieving information for transaction - ${txid}. Error Message - ${error.message}`
     );
+    res.status(503).send('Service Temporary Ununavailable');
```
Comment (rantan) [inline backend/actions/transaction_detail.js:38]:
```suggestion
    res.status(503).send('Service Temporary Unavailable');
```

File: backend/actions/transaction_detail.js [hunk 5]
```diff
@@ -58,6 +59,7 @@ app.get('/api/tx/:txid/rawData', async (req, res) => {
     logger.error(
       `Error retrieving rawdata for transaction - ${txid}. Error Message - ${error.message}`
     );
+    res.status(503).send('Service Temporary Ununavailable');
```
Comment (rantan) [inline backend/actions/transaction_detail.js:62]:
```suggestion
    res.status(503).send('Service Temporary Unavailable');
```

File: backend/actions/transaction_detail.js [hunk 6]
```diff
@@ -80,5 +82,6 @@ app.get('/api/tx/:txid/get', async (req, res) => {
     logger.error(
       `Error calling the method gettransaction for transaction - ${txid}. Error Message - ${error.message}`
     );
+    res.status(503).send('Service Temporary Ununavailable');
```
Comment (rantan) [inline backend/actions/transaction_detail.js:85]:
```suggestion
    res.status(503).send('Service Temporary Unavailable');
```

**Review by rantan — CHANGES_REQUESTED** [review]

File: backend/actions/block_detail.js (hunk 2 above)
Comment (Yamaguchi) [reply backend/actions/block_detail.js]: fix a0de50d8c92c84afb50780db8c2f157f3fd1115b

File: backend/actions/transaction_detail.js (hunk 4 above)
Comment (Yamaguchi) [reply backend/actions/transaction_detail.js]: fix a0de50d8c92c84afb50780db8c2f157f3fd1115b

File: backend/actions/transaction_detail.js (hunk 6 above)
Comment (Yamaguchi) [reply backend/actions/transaction_detail.js]: fix a0de50d8c92c84afb50780db8c2f157f3fd1115b

File: backend/actions/transaction_detail.js (hunk 5 above)
Comment (Yamaguchi) [reply backend/actions/transaction_detail.js]: fix a0de50d8c92c84afb50780db8c2f157f3fd1115b

**Review by rantan — APPROVED** [review]

**Review by azuchi — APPROVED** [review]

**Pull request merged**